sdk/core/azure-core/azure/core/pipeline/policies/universal.py

```diff
@@ -350,7 +350,10 @@ def deserialize_from_http_generics(cls, response):
         # Try to use content-type from headers if available
         content_type = None
         if response.content_type: # type: ignore
-            content_type = response.content_type[0].strip().lower() # type: ignore
+            try:
+                content_type = response.content_type.strip().lower() # type: ignore
+            except AttributeError:
+                content_type = response.content_type[0].strip().lower() # type: ignore
```

Comment:
My only suggestion really is that the try/except blocks here and above could go in their own functions e.g. `get_status_code`, `get_content_type` to keep this kind of conditioning based off implementation more hidden away

Reply:
Fixed!